src/StreamJsonRpc/ProxyGeneration.cs

```diff
@@ -133,66 +123,67 @@ internal static TypeInfo Get(Type contractInterface, ReadOnlySpan<Type> addition
             FieldBuilder calledMethodField = proxyTypeBuilder.DefineField("calledMethod", typeof(EventHandler<string>), FieldAttributes.Private);
             FieldBuilder marshaledObjectHandleField = proxyTypeBuilder.DefineField("marshaledObjectHandle", typeof(long?), fieldAttributes);
 
-            var ctorActions = new List<Action<ILGenerator>>();
-            foreach (TypeInfo contract in contractInterfaces)
+            if (FindAllOnThisAndOtherInterfaces(contractInterfaces, i => i.DeclaredProperties).FirstOrDefault() is PropertyInfo unsupportedProperty)
```

Comment:
Is this supposed to be `!FindAllOnThisAndOtherInterfaces` like it was before?

Right it now it looks like its just saying "if there are any properties on any of the contract interfaces then fail", unless I'm misinterpreting it?

Reply:
You interpret it correctly. It's supposed to fail if any interface has a property on it.

And that's what it did before. VerifySupported throws if you pass false to it. And the expression would evaluate to false if any properties were declared.

Reply:
Ah got it, I forgot StreamJsonRpc doesn't support properties